README.md

```diff
@@ -1,6 +1,91 @@
-# Podman Ansible Module
 
+Ansible Module for Podman
+=========================
+This module will allow to use Podman to manage the images.
 
-## License
+Requirements
+============
+* Python 3.6+
+* Ansible 2.8
 
-Ansible Podman module is licensed under Apache 2.0 license. See the [LICENSE](LICENSE) file for details.
+Installation
+============
+1. Install [Podman](https://github.com/containers/libpod)
```

Comment:
This would probably be a better link:
https://github.com/containers/libpod/blob/master/install.md

Reply:
Updated the link. Thank you.